rep-2000.rst

```diff
@@ -547,6 +547,126 @@ Build System Support:
+| rmw_fastrtps_dynamic_cpp | eProsima Fast-RTPS  | Tier 2        | All Platforms               | All Architectures                    |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+
+Middleware implementation support is dependent upon the platform support tier.
+For example a Tier 1 middleware implementation on a Tier 2 platform can only receive Tier 2 support.
+
+Minimum language requirements:
+
+- C++14
+- Python 3.7
+
+
+Dependency Requirements:
+
++-------------+-------------------------------------------------+--------------------------------------+
+|             |                        Required Support         |        Recommended Support           |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Package     |  Ubuntu Focal  |     MacOS**   |   Windows 10** | Debian Buster  | OpenEmbedded**      |
++=============+================+===============+================+================+=====================+
+| CMake       |     3.16.3     |     3.14.4    |     3.14.4     |      3.13.4    | 3.16.1 / 3.12.2**** |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| EmPY        |                                      3.3.2                                             |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Gazebo      |     11.0.0*    |     11.0.0    |      N/A       |      11.0.0*   |        N/A          |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Ignition    |             Citadel*           |      N/A       |     Citadel8   |        N/A          |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Ogre        |                                      1.10*                       |        N/A          |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| OpenCV      |     4.2.0      |     4.2.0     |     3.4.6*     |      3.2.0     |  4.1.0 / 3.2.0****  |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| OpenSSL     |     1.1.1d     |     1.1.1f    |     1.1.1f     |      1.1.1d    | 1.1.1d / 1.1.1b**** |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Poco        |     1.9.2      |     1.9.0     |     1.8.0*     |      1.9.0     |        1.9.4        |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Python      |     3.8.0      |     3.7.3     |     3.8.0      |      3.7.3     |  3.8.2 / 3.7.5****  |
```

Comment:
Looks like our dependencies are moving to Python 3.8 on homebrew

https://formulae.brew.sh/formula/pyqt
https://formulae.brew.sh/formula/numpy

Numpy we currently install from PyPI, but pyqt we get from homebrew. Maybe we can keep python 3.7 and use PyQt5 from PyPI

https://index.ros.org/doc/ros2/Installation/Eloquent/macOS-Install-Binary/
https://pypi.org/project/PyQt5/

Reply:
https://github.com/ros2/ci/pull/443

Reply:
https://github.com/ros2/ros2_documentation/pull/641

Reply:
Based on https://github.com/ros2/ros2_documentation/pull/663, I've bumped the version to 3.8.2 (2326172)